tests/filecheck/dialects/builtin/unrealized_conv_cast.mlir

```diff
@@ -1,16 +1,22 @@
-// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | filecheck %s
+// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | xdsl-opt | filecheck %s
```

Comment:
I'm not sure of the benefit of three `xdsl-opt`s

Reply:
For around-tripping printing and parsing of both generic and custom syntax.